src/json.test.ts

```diff
@@ -450,5 +443,150 @@ describe('json', () => {
         0,
       ]);
     });
+
+    it('should return true or false for validity depending on the test scenario from ECMA TC39 (test262)', () => {
+      // This test will perform a series of validation assertions.
+      // These tests are taken from ECMA TC39 (test262) test scenarios used
+      // for testing the JSON.stringify function.
+      // https://github.com/tc39/test262/tree/main/test/built-ins/JSON/stringify
+
+      // Value: array circular
+      const direct: any = [];
```

Comment:
Yes, sure. I would like to avoid `any` too. But, this is the case where I'm converting JavaScript tests to TypeScript tests which are directly testing bad things that should not be used in most cases.

And for that particular one, I'm getting annoying error every time if I use something that's not `any`.

![Screenshot 2022-07-07 at 16 33 32](https://user-images.githubusercontent.com/13301024/177799701-4f606729-52a8-4f10-960e-170b973a9d16.png)

So, other than `any` doesn't let the tests to pass.

Reply:
Looks like you're using `const direct: unknown` rather than `const direct: unknown[]`? There's a difference between those.